SPECS-SIGNED/mstflint-driver-signed/mstflint-signed.spec

```diff
@@ -0,0 +1,35 @@
+%define mstflint_unsigned_name mstflint-kernel
+
+Summary:        Mellanox firmware burning tool
+Name:           %{mstflint_unsigned_name}-signed
+Version:        4.21.0
+Release:        4%{?dist}
```

Comment:
This reminds me: we probably want to "entangle" this spec with `mstflint.spec`. The idea is to make it harder to miss propagating the updates made in `mstflint.spec` to this spec. If anything related to `mstflint-kernel` ever changes in `mstflint.spec`, then we'll need to bump the `Release` number in this spec, so that we produce a package with a new version and the changes actually get published. The "entangled" specs get validated by our PR checks, so we block them and avoid these mistakes.

Updating the check should only require adding another `frozenset` inside `toolkit/scripts/check_entangled_specs.py`. Similar to the `kernel-hci.spec` and `kernel-hci-signed.spec` pair.

**NOTE**: any changes inside `mstflint.spec` will then require a change in this one as well. Even the ones, which don't affect the `*-kernel` subpackage. One way to avoid that would be to split the `mstflint-kernel` package into a completely separate spec file. I'll need to think a bit more about the potential pros and cons, though.

Reply:
I don't think there would be frequent changes in the spec and both are working on the same source tarball. If there is any value in separating them, we can do it later.

Reply:
Got it, that sounds good to me.

We'd still want to entangle the specs. The "signed" and regular specs split is just a way to produce packages, which need some of their components to be signed, but logically they are viewed as a single spec, so their versions and releases should be updated in lockstep.

Reply:
Done! Thanks!